ReadModelTest/ModelGeneratorTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using EventReader.Event;
+using EventReader.Read;
+using ReadModel;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace ReadModelTest
+{
+    public class ModelGeneratorTest
+    {
+        [Fact]
+        public void PopulateModel()
```

Comment:
This test is a good indication that we can improve our abstractions to reduce coupling.  We want to get to the point where our test simply instantiates a few events and passes them into the model, and we assert on the model's content.  We shouldn't need to touch anything to do with sql to achieve that.